zlibWrapper/zstd_zlibwrapper.c

```diff
@@ -220,7 +220,7 @@ ZEXTERN int ZEXPORT z_deflate OF((z_streamp strm, int flush))
         strm->next_out += dstCapacity;
         strm->total_out += dstCapacity;
         strm->avail_out -= dstCapacity;
-        if (flush == Z_FINISH && bytesLeft == 0) return Z_STREAM_END;
+        if (bytesLeft == 0) return Z_STREAM_END;
```

Comment:
cc @inikep .
There is a large update of zlib_wrapper ongoing.
I'm unsure if these change remains relevant after the update.